libs/httpproxy/proxy-camel-route/src/main/java/org/codice/proxy/http/HttpProxyServiceImpl.java

```diff
@@ -184,6 +187,7 @@ public void configure() throws Exception {
         LOGGER.debug("Started proxy route at servlet endpoint: {}, routing to: {}",
                 endpointName,
                 targetUri);
+        endpointIds.add(endpointName);
```

Comment:
Shouldn't we also be removing these endpoints from this array when the stop(...) method is called?